addon-test-support/-private/get-test-loader.js

```diff
@@ -1,12 +1,19 @@
 /* globals require, requirejs */
 
+/**
+ * Returns ember-exam-qunit-test-loader or ember-exam-mocha-test-loader
+ *
+ * @export
+ * @returns {Object}
+ */
 export default function getTestLoader() {
-  let testLoaderModulePath = 'ember-cli-test-loader/test-support/index';
-
-  if (!requirejs.entries[testLoaderModulePath]) {
-    testLoaderModulePath = 'ember-cli/test-loader';
+  if (requirejs.entries['ember-qunit/test-loader']) {
+    const TestLoaderModule = require('./ember-exam-qunit-test-loader');
+    return TestLoaderModule['default'];
+  } else if (requirejs.entries['ember-mocha/test-loader']) {
+    const TestLoaderModule = require('./ember-exam-mocha-test-loader');
+    return TestLoaderModule['default'];
   }
 
-  const TestLoaderModule = require(testLoaderModulePath);
-  return TestLoaderModule['default'];
-}
+  throw new Error('Cannot find known test loader.');
```

Comment:
Would this error message be more accurate?

`throw new Error('Unable to find a suitable test loader. You should ensure that one of \`ember-qunit\` or \`ember-mocha\` are added as dependencies.');`

Reply:
That would. Fixed